README.md

```diff
@@ -1,3 +1,17 @@
 # Validator kit
 
-TODO: add docs here how to use the validator kit
+## Overview
+
+This repository contains the validator kit for Mezo chain. The validator kit is a collection of tools and documentation to help you run a validator node on Mezo chain. There are three main components of the validator kit:
```

Comment:
Remember that there will be a third option: Helm chart. We should have a PR from the Boar team soon.